### PR TITLE
Add gen3-release label to prs which update ci envs

### PR DESCRIPTION
**Pull request by jingh8**

### CODE REVIEW

File: gen3release-sdk/gen3release/gith/git.py [hunk 1]
```diff
@@ -155,6 +155,7 @@ def create_pull_request_apply(
         the_pr = github_client.create_pull(
             title=pr_title, body=commit_msg, head=branch_name, base="master"
         )
+        the_pr.add_to_labels("gen3-release")
```
Comment (atharvar28) [inline gen3release-sdk/gen3release/gith/git.py:158]: We dont want to do this - as we use `apply` command while creating PRs for cdis-manifest release PR

Comment (haraprasadj) [reply gen3release-sdk/gen3release/gith/git.py:158]: I think we will have to add an argument for labels and use that in the job 🤔

**Review by atharvar28 — APPROVED** [review]
🚀

**Review by haraprasadj — APPROVED** [review]
+1

**Pull request merged**